packages/loader/container-loader/src/connectionManager.ts

```diff
@@ -647,19 +648,19 @@ export class ConnectionManager implements IConnectionManager {
 
 				const waitStartTime = performance.now();
 				const retryDelayFromError = getRetryDelayFromError(origError);
-				if (retryDelayFromError !== undefined) {
-					// If the error told us to wait, then we wait.
-					this.props.reconnectionDelayHandler(retryDelayFromError, origError);
+				if (retryDelayFromError !== undefined || globalThis.navigator?.onLine !== false) {
```

Comment:
If we take "ELSE" branch, then delayMs would not be changed, right? vs. previously it was always doubled.
I think it will reintroduce old issue where if service does not supply retryAfter, then client will keep coming back immediately.

Reply:
It will be doubled. We are calculating wait time based on error and then parallelly calculating our value and taking max of both.